src/analytics.js

```diff
@@ -164,6 +164,7 @@ define(function()
             if (gaAvailable === true && gaEventOptions.eventCategory != '' && gaEventOptions.eventAction != '') {
                 $(gaEventOptions).each(function(index, options) {
                     var sendGATracking = true;
+                    var trackingGAObjectName = ga.getAll()[0].get('name');
```

Comment:
Hi Pete, this line seems quite heavily chained without any protection. This is prone to breaking if ga.getAll() fails or [0] is undefined or .get('name') fails. Is there a default name we can use incase this fails eg: 

```
var trackingGAObjectName;
try {
        trackingGAObjectName  = ga.getAll()[0].get('name');
} catch(e) {
        trackingGAObjectName = 'FALLBACK NAME';
}
```